client/src/components/section1/Section1.jsx

```diff
@@ -1,17 +1,113 @@
 import React, { Component } from 'react';
+import styled from 'styled-components';
 
 class Section1 extends Component {
     constructor(props) {
         super(props)
     }
 
+
     render() {
+        let randomIdx = Math.floor(Math.random() * 100)
+        
         return(
             <div>
-                section1
+                <Container >
+                    <Left >
+                        <Title>
+                            {this.props.house.length > 0 && this.props.house[randomIdx].title}
+                        </Title>
+                        <Location>
+                            {this.props.house.length > 0 && this.props.house[randomIdx].location}
+                        </Location>
+                    </ Left>
+                    <Right >
+                        <Pic src={this.props.house.length > 0 && this.props.house[randomIdx].imageURL} ></Pic>
+                        {/* <Pic src='http://lorempixel.com/640/480/people/'></Pic> */}
+                        <Host >{this.props.house.length > 0 && this.props.house[randomIdx].hostName}</Host>
+                    </ Right>
+
+                    
+                </Container>
+                <Line2 />
             </div>
         )
     }
 }
 
+const Line = styled.div`
+    border-bottom-width: var(--border-rule-border-width, 1px) !important;
+    border-bottom-color: var(--color-divider, #EBEBEB) !important;
+    border-bottom-style: solid !important;
+`;
```

Comment:
Why do these need `!important`?

And where are these CSS variables being used?